themes/ddbasic/template.php

```diff
@@ -1081,17 +988,16 @@ function ddbasic_date_display_range($variables) {
  * Ting search pager.
  */
 function ddbasic_ting_search_pager($variables) {
-
-  if(!empty($_GET['size'])) {
+  if (!empty($_GET['size'])) {
```

Comment:
The drupal_get_query_parameters is primarily used for filtering query parameters, and passing the rest on. In this code we use a specific key.